SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2260,7 +2257,14 @@ void USpatialReceiver::ResolveIncomingOperations(UObject* Object, const FUnrealO
 			continue;
 		}
 
-		FSpatialObjectRepState* RepState = DependentChannel->ObjectReferenceMap.Find(ChannelObjectIter->Value);
+		RepNotifyCalls.Push({ DependentChannel, ChannelObjectIter->Value });
+	}
+
+	for (const auto& RepNotifyCall : RepNotifyCalls)
+	{
+		USpatialActorChannel* DependentChannel = RepNotifyCall.Channel;
+		UObject* ReplicatingObject = RepNotifyCall.Object.Get();
+		FSpatialObjectRepState* RepState = DependentChannel->ObjectReferenceMap.Find(RepNotifyCall.Object);
```

Comment:
Concerned that the RepGraph can do a whole load of weird things with channels, like closing/(deleting?) (not sure where these are destroyed/deleted?). 
There is a weird path in worker->worker replication where a RepNotify can remove itself or other actors (potentially ahead in this array)

Reply:
Not sure if this is actually an issue or not, the channel might be closed but not sure whether the pointer is still valid

Reply:
This is another issue though, think as a continuation of the workaround this also works well

Reply:
Hmm, if I remember correctly, ActorChannels are pooled, so potentially I guess if a channel is closed and reused, we could be resolving completely different channels than we were originally doing.